src/realm/object-store/schema.cpp

```diff
@@ -348,20 +366,42 @@ std::vector<SchemaChange> Schema::compare(Schema const& target_schema, SchemaMod
     return changes;
 }
 
-void Schema::copy_keys_from(realm::Schema const& other) noexcept
+void Schema::copy_keys_from(realm::Schema const& other, bool allow_complete_schema_view) noexcept
```

Comment:
The name `allow_complete_schema_view` in all these functions is weird. "Allow" doesn't really make sense and it doesn't really indicate what setting the bool to true does.